src/custom/constants/index.ts

```diff
@@ -16,6 +17,16 @@ export * from '@src/constants/index'
 
 export const PRODUCTION_URL = 'cow.trade'
 
+const DISABLED_WALLETS = /^(?:WALLET_LINK|COINBASE_LINK|WALLET_CONNECT|FORTMATIC|Portis)$/
```

Comment:
I would make this case insentive, just in case

Reply:
done